app/javascript/controllers/recipe_form_styling_controller.js

```diff
@@ -0,0 +1,18 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
+    connect() {
+
+        const tx = document.getElementsByTagName("textarea");
+
+        for (let i = 0; i < tx.length; i++) {
+            tx[i].setAttribute('style', `height: ${tx[i]?.id === 'recipe_description' || tx[i]?.id === 'recipe_notes' ? '70px' : '230px'}`);
+            tx[i].addEventListener("input", OnInput, false);
+        }
+
+        function OnInput() {
+            this.style.height = this?.id === 'recipe_description' || this?.id === 'recipe_notes' ? "70px" : "230px";
+            this.style.height = (this.scrollHeight) + "px";
+        }
+    }
```

Comment:
Here's a suggestion that addresses the following issues
- Stimulus controllers know what elements they are attached to
- Stimulus provides a way to manage event listeners via actions
- The initial height of the element can be determined roughly using `offsetHeight`
- If you add a few pixels to `scrollHeight`, a scrollbar will not be added to the textarea

```suggestion
  connect() {
    this.initialHeight = this.element.offsetHeight
    this.resizeElement(this.element)
  }

  resize(event) {
    this.resizeElement(event.target)
  }

  resizeElement(element) {
    element.style.height = this.initialHeight + "px";
    element.style.height = element.scrollHeight + 3 + "px";
  }
```